fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```
@@ -1283,6 +1284,10 @@ public void updatePartitionTabletMeta(Database db,
         db.readLock();
         try {
             OlapTable olapTable = (OlapTable) db.getTable(tableName);
+            // LakeTable not support update tablet meta
+            if (olapTable instanceof LakeTable) {
+                return;
```

I think it is better to throw unsupported exception.